### PR TITLE
adds the new ratio of user defined vs OCP tags

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to main and 4.12
- [Jira](https://issues.redhat.com/browse/CFE-592)
- [Preview](https://50969--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-configuration-parameters_installing-aws-customizations)
- @TrilokGeer @yunjiang29 ptal.

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -766,11 +766,16 @@ host must trust the certificate.
 |`platform.aws.userTags`
 |A map of keys and values that the installation program adds as tags to all resources that it creates.
 |Any valid YAML map, such as key value pairs in the `<key>: <value>` format. For more information about AWS tags, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources] in the AWS documentation.
+[NOTE]
+====
+You can add upto 25 user defined tags during installation. The ratio is 25:25 that is user defined tags: reserved {product-title} tags.
```
Comment (TrilokGeer) [inline modules/installation-configuration-parameters.adoc:771]: It is not a ratio. It should be "Remaining tags are reserved for Open Shift"

Comment (Amrita42) [reply modules/installation-configuration-parameters.adoc:771]: I have specified the tags , ptal

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -767,10 +767,16 @@ host must trust the certificate.
 |A map of keys and values that the installation program adds as tags to all resources that it creates.
 |Any valid YAML map, such as key value pairs in the `<key>: <value>` format. For more information about AWS tags, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources] in the AWS documentation.
 
+[NOTE]
```
Comment (nalhadef) [inline modules/installation-configuration-parameters.adoc:770]: Please add a content type at the top of this document.

File: modules/installation-configuration-parameters.adoc [hunk 3]
```diff
@@ -767,10 +767,16 @@ host must trust the certificate.
...
+====
+You can add upto 25 user defined tags during installation. The remaining 25 tags are reserved for {product-title}.
```
Comment (nalhadef) [inline modules/installation-configuration-parameters.adoc:772]: - Replace "upto" with "up to."
- The third cell for this note expands the column off screen.
image

File: modules/installation-configuration-parameters.adoc (hunk 2 above)
Comment (Amrita42) [reply modules/installation-configuration-parameters.adoc:770]: @nalhadef content type is there at line 205

File: modules/installation-configuration-parameters.adoc (hunk 3 above)
Comment (Amrita42) [reply modules/installation-configuration-parameters.adoc:772]: there is a scrolling bar at the bottom of the table

File: modules/installation-configuration-parameters.adoc [hunk 4]
```diff
@@ -767,10 +767,16 @@ host must trust the certificate.
...
+[NOTE]
+====
+You can add up to 25 user defined tags during installation. The remaining 25 tags are reserved for {product-title}.
+====
+
 |`platform.aws.propagateUserTags`
 | A flag that directs in-cluster Operators to include the specified user tags in the tags of the AWS resources that the Operators create.
 | Boolean values, for example `true` or `false`.
 
+
```
Comment (aireilly) [inline modules/installation-configuration-parameters.adoc:779]: This line break is not needed

File: modules/installation-configuration-parameters.adoc [hunk 5]
```diff
@@ -767,10 +767,16 @@ host must trust the certificate.
 |A map of keys and values that the installation program adds as tags to all resources that it creates.
 |Any valid YAML map, such as key value pairs in the `<key>: <value>` format. For more information about AWS tags, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources] in the AWS documentation.
```
Comment (aireilly) [inline modules/installation-configuration-parameters.adoc:768]: Shouldn't this cell require an `a` modifier? https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#a-operator

**Review by aireilly — COMMENTED** [review]
Changes LGTM. However, I'm concerned that this table is not very useable at all on the docs.openshift site build. The scroll bar is at the bottom of the table making it very difficult to read because you have to scroll up and down and left and right.

File: modules/installation-configuration-parameters.adoc (hunk 5 above)
Comment (aireilly) [reply modules/installation-configuration-parameters.adoc:768]: I'll raise it as a separate issue

Comment (Amrita42) [reply modules/installation-configuration-parameters.adoc:768]: Yes thank you :)

**Comment by nalhadef** [comment]
LGTM, @Amrita42!

**Comment by yunjiang29** [comment]
lgtm.

**Pull request merged**